h/accounts/views.py

```diff
@@ -587,6 +587,16 @@ def reset_password_link(request, reset_code):
     return request.route_url('reset_password_with_code', code=reset_code)
 
 
+@view_config(route_name='dismiss_sidebar_tutorial', request_method='POST',
+             renderer='json')
+def dismiss_sidebar_tutorial(request):
+    if request.authenticated_userid is None:
+        raise accounts.JSONError()
+    else:
```

Comment:
I'm going to agree with nick here on style. In client code the recommended style is:

```js
if (condition) {
  // early exit
   throw SomeError() // or 'return x';
}
// happy path
```

It avoids unnecessary indentation.